* Every os that doesn't match any of the previous cases, and has
  /etc/netconfig (like IRIX) might bark that /usr/xpg4/bin/grep is not
  available.  For Prefix we prefer to use our own grep anyway.

http://bugs.python.org/issue7716
committed in r87698 (3.2), r87699 (3.1) and r87700 (2.7)

--- configure.in
+++ configure.in
@@ -2116,7 +2116,7 @@
 			;;
 		solaris)
 			if test -f /etc/netconfig; then
-                          if /usr/xpg4/bin/grep -q tcp6 /etc/netconfig; then
+                          if $GREP -q tcp6 /etc/netconfig; then
 				ipv6type=$i
 				ipv6trylibc=yes
                           fi
